fw/http.c

```diff
@@ -6072,24 +6088,23 @@ tfw_http_req_process(TfwConn *conn, TfwStream *stream, struct sk_buff *skb,
 
 	case TFW_HTTP_SESS_VIOLATE:
 		TFW_INC_STAT_BH(clnt.msgs_filtout);
-		return tfw_http_req_parse_block(req, 503, NULL,
+		return tfw_http_req_parse_block(req, 403, NULL,
 						HTTP2_ECODE_PROTO);
 
 	case TFW_HTTP_SESS_JS_NOT_SUPPORTED:
 		/*
-		 * Requested resource can't be challenged, forward all pending
-		 * responses and close the connection to allow client to recover.
+		 * Requested resource can't be challenged.
 		 */
-		TFW_INC_STAT_BH(clnt.msgs_filtout);
-		return tfw_http_req_parse_block(req, 503,
+		*splitted = skb;
+		return tfw_http_req_parse_drop(req, 403,
 				"request dropped: can't send JS challenge"
 				" since a non-challengeable resource" 
 				" (e.g. image) was requested",
 				HTTP2_ECODE_PROTO);
```

Comment:
I think `msg` should be NULL to avoid https://github.com/tempesta-tech/tempesta/issues/1102#issuecomment-672109845 . It makes sense to print this message on a debug level 1 to debug some problems. Basically, this is pretty normal condition, e.g. some 3rd-party resource links an image from a protected resource or browser takes an HTML document from the cache and fetches a CSS or other non-challengeable resource.